src/coreclr/src/jit/error.h

```diff
@@ -153,24 +154,17 @@ extern void RecordNowayAssertGlobal(const char* filename, unsigned line, const c
 #define unreached() noWayAssertBody()
 
 #define NOWAY_MSG(msg) noWayAssertBodyConditional(NOWAY_ASSERT_BODY_ARGUMENTS)
+#define NOWAY_MSG_FILE_AND_LINE(msg, file, line) noWayAssertBodyConditional(NOWAY_ASSERT_BODY_ARGUMENTS)
```

Comment:
I guess `file` and `line` are ignored here, in favor of `__FILE__`, `__LINE__`

Reply:
Yes. Well, sortof. This is the release portion of the ifdef, where __FILE__ and __LINE__ aren't used at all. The goal is to match the overall behavior of what existed before.

Reply:
In particular, non-altjits don't put in file/line number stuff, but chk/dbg will.